src/transformers/__init__.py

```diff
@@ -1503,6 +1511,16 @@
             "OPTPreTrainedModel",
         ]
     )
+    _import_structure["models.owlvit"].extend(
+        [
+            "OWLVIT_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "OwlViTModel",
+            "OwlViTPreTrainedModel",
+            "OwlViTTextModel",
+            "OwlViTVisionModel",
+            "OwlViTForObjectDetection",
+        ]
+    )
```

Comment:
These imports are missing in the second part of the init in the TYPE_CHECKING section.

Reply:
@sgugger yes, I'm adding the imports now